### ISSUE TITLE
There is no "New Kata" button

### ISSUE THREAD

**Issue by DenisDziganchuk** [codewars/codewars.com#2339]
When I wanted to check my katas and press the "New Kata" button, there was no button.
Please help me.

Screenshot:
![Screen Shot 2021-03-10 at 3 25 21 PM](https://user-images.githubusercontent.com/73368785/110711840-0cdff680-81b5-11eb-98cd-0c2401a71621.png)

**Comment by hobovsky** [comment]
I think that some of your privileges were revoked as a result of [some reports](https://github.com/codewars/codewars.com/issues/1493#issuecomment-793235978).

**Comment by kazk** [comment]
You (and some of your clan members) have lost the privilege to author kata because:

1. you kept ignoring feedback and published many low-quality kata
2. your clan members were up voting each other's kata ignoring the community standards

https://github.com/codewars/codewars.com/issues/1493#issuecomment-793235978

I'm willing to give you another chance if you promise to change your behavior.

Note that preventing automatic retiring by collecting few votes doesn't mean much because I can always retire them manually, even after approving.

---

Also, if I remember correctly, someone from your clan emailed us asking to approve their teacher's kata. Codewars is not for hosting your classroom assignments. We have a separate service for education where you can create any challenge you want. See https://www.qualified.io/for-education

**Comment by DenisDziganchuk** [comment]
Sorry, we didn't know that. We are learning coding and making katas. We showed each other our katas to practice. We won't do that again. Please forgive us. Thank you for your answer and advice.

**Comment by Koawen** [comment]
Hey, Can I please ask if I made a Kata, and one of my clan members upvotes it, Not because He wants to help me but he actually Likes it does it still Breaks the rules.

**Comment by kazk** [comment]
Done. Please make sure all of your friends know as well. If this happens again, you'll lose it permanently. I'll also consider banning you guys from Codewars depending on the severity.

> Hey, Can I please ask if I made a Kata, and one of my clan members upvotes it, Not because He wants to help me but he actually Likes it does it still Breaks the rules.

No. There's no way to know "why" they up voted, so that alone doesn't mean anything. I can only act when that leads actual problems.

You should be fine as long as you're cooperative with the rest of the community (don't ignore their feedback and keep publishing new kata), and you're not up voting just because your friend made it.

Basically, just don't push too much to get your friends'/teacher's kata approved and don't annoy other users with it. You can effectively bypass the beta process by having many friends blindly up voting, but it's pointless because I can unapprove them. It's just annoying.

**Comment by kazk** [comment]
@DenisDziganchuk

> You should be fine as long as you're cooperative with the rest of the community (**don't ignore their feedback and keep publishing new kata**) 

You wrote you had read the documentation.

![image](https://user-images.githubusercontent.com/639336/111026495-6146ca00-839f-11eb-96f2-c20db5dbbc51.png)

It clearly says not to do what you had published.

> These things are what you **shouldn't do in bugfix katas**:
> 
> - A "pseudo-code" program written in unknown, self-invented syntax that will not even compile at all
> - Highly obfuscated or ugly programs
> 
> Basically, if a bugfix kata features code that is so smelly rewriting it from scratch would be more pleasant and faster, it stops being a bugfix kata and becomes a regular kata.

And that was your **19**th kata.

---

> Sorry, we didn't know that. ... We won't do that again.

You're clearly just saying this without thinking, just like how you've been with the community. I regret spending the time to reply to you. I think we (the Codewars community) gave you more than enough opportunities to improve.
You no longer have the privilege to author. The decision is final.

**Comment by DenisDziganchuk** [comment]
Please, can you give me one last chance?  I promise to not make katas frequently and to read the guide before making katas. I forgot that bugfix katas should compile.
